packages/@lwc/integration-karma/test/rendering/issue-3396/x/test/test.html

```diff
@@ -0,0 +1,5 @@
+<template>
+    <div lwc:if={show}>1</div>
```

Comment:
More than one node inside here should repro any issues with multiple `content` nodes.

Reply:
Added this test case to `sequential-reordering` to keep the test and the submitted repro similar.